### DISCUSSION TITLE
Versions 1.0.1 to 1.0.4 marked as pre-release

### DISCUSSION THREAD

**Discussion by AxeldeWater** [JunkFood02/Seal#126] [General]
I learnt that versions 1.0.1 to 1.0.4 are marked as pre-releases and not official releases. Why is this?

**Comment by JunkFood02** [comment]
No specific reason tho. It's just that the features are updated so quickly that sometimes I lack confidence in the stability. 😂 You can still use the latest version without much worry.

**Reply by AxeldeWater** [reply]
Well it kinda automatically updates for most people on F-Droid so it would be seen as an official release by most people anyway, so I don't think marking it as a pre-release has much value.

**Reply by JunkFood02** [reply]
Thanks for the info. I don't know much about F-Droid. I will be more aware of the stability in the next releases. If I release a preview in the future, I will limit the F-Droid version to a stable one. 😌
